src/main/resources/db/migration/V0__create_tables_schema.sql

```diff
@@ -0,0 +1,11 @@
+set search_path to shop;
+
+CREATE EXTENSION "uuid-ossp";
+
+create table customer (id uuid not null default uuid_generate_v4(), email_address varchar(255), first_name varchar(255), last_name varchar(255), password varchar(255), username varchar(255), primary key (id));
```

Comment:
Style your SQL files. The SQL-reserved keywords need to be written in uppercase. Have a look here https://www.sqlstyle.guide/.